tests/unit/test_ruleparser.py

```diff
@@ -0,0 +1,103 @@
+from unittest import TestCase
```

Comment:
I don't think this file belongs here.

Reply:
I just moved it here. For some reason it was located in `esque/ruleparser/tests/unit`. Don't know why github doesn't recognize that.

Reply:
You have to delete `esque/ruleparser/tests`

Reply:
ah goddamnit